inspect.lua

```diff
@@ -305,6 +307,7 @@ function inspect.inspect(root, options)
    local newline = options.newline or '\n'
    local indent = options.indent or '  '
    local process = options.process
+   local print_reference = options.print_reference or true
```

Comment:
print_reference will always be true because `or true` converts false and nil to true.

Consider: `options.print_reference or options.print_reference == nil`